docs/developer-docs/cost-estimations-and-examples.md

```diff
@@ -0,0 +1,173 @@
+# Cycles cost estimations and examples
+
+
+## Overview
+
+To get a rough estimate of how much your project may cost, below are a few common project architectures and their estimated monthly cost in cycles. These estimates are broken down into the following  groups:
+
+
+- **Messaging**: A message transmission refers to calls that are made to a canister's methods. Messaging costs depend on the type of call being sent (query, update, inter-canister, etc), the size of the message's request and response bytes, and the total amount of messages a canister sends and receives.
+
+- **Execution**: This refers to cycles charged during execution of message and tasks. There is a fixed fee for starting execution and fee per each executed instruction.
+
+- **Storage**: This refers to cycles charged for canister storage. Note that it includes both the Wasm memory and the stable memory.
+
+- **Special features**: Special feature transmissions include HTTPS outcalls, transmissions using the [Bitcoin API](./integrations/bitcoin/bitcoin-how-it-works.md), and transmissions using the [chain-key signing API](./integrations/t-ecdsa/t-ecdsa-how-it-works.md).
+
+## Units and fiat value
+
+The price of cycles is fixed against the price of [XDR](/docs/current/references/glossary#xdr), where **1 trillion cycles equals 1 XDR**.
+
+This documentation will use the following units to measure and calculate the number of cycles and their associated fiat value:
+
+
+| Abbr. | Name | In numbers | Cycles XDR value | Cycles USD value |
+|---------------------------------------------------------------------------|
+| T | Trillion | 1_000_000_000_000 | 1 | 1.34 |
+| B | Billion | 1_000_000_000 | 0.001 | 0.00134 |
+| M | Million | 1_000_000 | 0.000001 | 0.00000134 |
+| k | Thousand | 1_000 | 0.000000001 | 0.00000000134 |
+| – | (one) | 1 | 0.000000000001 | 0.00000000000134|
+
+
+## How is the number of cycles charged to a canister estimated?
```

Comment:
Would it make sense to move this to the end, such that the user doesn't get overwhelmed with this list?

Reply:
Moved to the end of the page